builders/tools/precompiles/democracy.md

```diff
@@ -0,0 +1,203 @@
+ - **propose**(*bytes32* proposal_hash, *uint256* value) — submit a proposal by providing a hash and the number of tokens to lock 
+ - **second**(*uint256* prop_index, *uint256* seconds_upper_bound) — second a proposal by providing the proposal index and a number greater than or equal to the number of existing seconds for this proposal (necessary to calculate the weight of the call). An amount is not needed because seconds require the same amount the original proposer locked 
+ - **standard_vote**(*uint256* ref_index, *bool* aye, *uint256* vote_amount, *uint256* conviction) — vote in a referendum by providing the proposal index, the vote direction (`true` is a vote to enact the proposal, `false` is a vote to keep the status quo), the number of tokens to lock, and the conviction. Conviction is an integer from `0` to `6` where `0` is no lock time and `6` is the maximum lock time
+ - **remove_vote**(*uint256* ref_index) — this method is used to remove a vote for a referendum before clearing expired democracy locks. Note, this cannot be used to revoke or cancel a vote while a proposal is being voted on 
+ - **delegate**(*address* representative, *uint256* candidateCount, *uint256* amount) — delegate voting power to another account by specifying an account to whom the vote shall be delegated, a conviction factor which is used for all delegated votes, and the number of tokens to delegate.
+ - **un_delegate**() — a method called by the delegator to undelegate voting power. Tokens are eligible to be unlocked once the conviction period specified by the original delegation has elapsed
+ - **unlock**(*address* target) — unlock tokens that have an expired lock. You MUST call **remove_vote** for each proposal with tokens locked you seek to unlock prior to calling **unlock**, otherwise tokens will remain locked. This function may be called by any account
+ - **note_preimage**(*bytes* encoded_proposal) — Registers a preimage for an upcoming proposal. This doesn't require the proposal to be in the dispatch queue but does require a deposit which is returned once enacted
+ - **note_imminent_preimage**(*bytes* encoded_proposal) — Register the preimage for an upcoming proposal. This requires the proposal to be in the dispatch queue. No deposit is needed. When this call is successful, i.e. the preimage has not been uploaded before and matches some imminent proposal, no fee is paid
+
+The interface also includes the following functions which are not currently supported but may be supported in the future:
+
+  - **ongoing_referendum_info**(*uint256* ref_index) — read-only function that returns the details of the specified ongoing referendum in the form of a tuple that includes the following:
+    1. block in which the referendum ended (*uint256*)
+    2. the proposal hash (*bytes32*)
+    3. the biasing mechanism where 0 is SuperMajorityApprove, 1 is SuperMajorityAgainst, 2 is SimpleMajority (*uint256*)
```

Comment:
maybe add links to Polkadot for the SuperMajority* variables

https://wiki.polkadot.network/docs/learn-governance#super-majority-approve

I wasn't sure what they were so I googled it haha